### DISCUSSION TITLE
Clarification on some options ("googlevisibility", "metric" and "Bored Waiting?")

### DISCUSSION THREAD

**Discussion by shalak** [Stirling-Tools/Stirling-PDF#338] [Q&A] [answered]
Hello!

Where can I find clarification of the following options:

- `system.googlevisibility` - what does this do?
- `metrics.enabled` - what is the "metrics" api? I don't see anything with "metrics" in swagger
- in the GUI, there's a `Bored Waiting?` checkbox - what is it for?

**Comment by Frooodle** [comment, marked as answer]
Google visibility is a toggle for the robots.txt file which tells Google and other search engines to scan your webpage or not 

Basically to turn Google visibility on or off

Metrics api turns on and off all /api/ endpoints 
There are ones which exist to grab total queries and requests etc
They should be shown in swagger 


Bored Waiting is just a easter egg you can ignore it :)

**Reply by shalak** [reply]
Thanks! I've taken a liberty of pushing a [PR for updating the docs to reflect the first two items](https://github.com/Frooodle/Stirling-PDF/pull/341).
